https://github.com/wwylele/teakra/commit/5862fd000353b8086b5be799c0a7f3ee1b2fe971.patch

From 5862fd000353b8086b5be799c0a7f3ee1b2fe971 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Mon, 14 Jan 2019 22:21:41 +0000
Subject: [PATCH] Align TestCase on 32-bit x86

src/test.h:37:1: fatal error: static_assert failed
static_assert(sizeof(TestCase) == 4312);
^             ~~~~~~~~~~~~~~~~~~~~~~~~
---
 src/test.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/teakra/src/test.h b/src/teakra/src/test.h
index e8c8742..869f8d2 100644
--- a/src/teakra/src/test.h
+++ b/src/teakra/src/test.h
@@ -29,7 +29,7 @@ struct State {
 
 static_assert(std::is_trivially_copyable_v<State>);
 
-struct TestCase {
+struct alignas(8) TestCase {
     State before, after;
     u16 opcode, expand;
 };

